### ISSUE TITLE
a failing case 

### ISSUE THREAD

**Issue by zenglian** [HexarA/Json2Pojo#11]
This case is from Spark's rest API,  a get of environment (removed more items in the array)

```
{
  "runtime" : {
    "javaVersion" : "1.8.0_144 (Oracle Corporation)",
    "javaHome" : "/usr/lib/jvm/jdk1.8.0_144/jre",
    "scalaVersion" : "version 2.11.8"
  },
  "sparkProperties" : [ 
  	[ "spark.serializer", "org.apache.spark.serializer.KryoSerializer" ],
  	[ "spark.sql.warehouse.dir", "hdfs://master:9000/spark/sql/warehouse" ]],
  "systemProperties" : [ [ "java.io.tmpdir", "/tmp" ], [ "line.separator", "\n" ], [ "path.separator", ":" ]],
  "classpathEntries" : [ [ "/usr/local/spark/jars/commons-lang-2.6.jar", "System Classpath" ]]
}
```

Generated class is. The other 3 array fields are expected but missing.
```
public class Env {

    @SerializedName("runtime")
    private Runtime mRuntime;

    public Runtime getRuntime() {
        return mRuntime;
    }

    public void setRuntime(Runtime runtime) {
        mRuntime = runtime;
    }

}
```

**Comment by HexarA** [comment]
Thanks for opening this issue. It appears that Json2Pojo has issues parsing arrays of arrays in source JSON text. I suspect the right thing to do in this case would be to generate fields such as `private List<List<String>> mSparkProperties`, etc. I'll see what I can do to fix this relatively soon.

**Comment by HexarA** [comment]
Should be fixed in 1.0.9; please update and let me know if that is not the case.
